src/App.css

```diff
@@ -2,4 +2,205 @@ body {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  background: #ccc;
+}
+
+.navbar {
+  display: flex;
+  align-items: center;
+
+  /* justify-content: space-between; */
+  background-color: #fff;
+  padding: 10px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
+
+.logo {
+  font-size: 24px;
+  font-weight: bold;
+  color: rgb(91, 91, 247);
```

Comment:
- [ ] the color you are using for your design is not what used in the original template, `#0290ff`, please make sure you update this